lib/apollo-federation/tracing/tracer.rb

```diff
@@ -116,16 +116,18 @@ def self.execute_field(data, &block)
 
         end_time_nanos = Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond)
 
-        # interpreter runtime
+        # legacy runtime
         if data.include?(:context)
           path = context.path
           field_name = context.field.graphql_name
           field_type = context.field.type.to_s
           parent_type = context.parent_type.graphql_name
-        else # legacy runtime
+        else # interpreter runtime
```

Comment:
I believe we got the legacy and new interpreter the wrong way round. I get into this branch